java/core/src/main/java/com/whylogs/core/ColumnProfile.java

```diff
@@ -0,0 +1,80 @@
+package com.whylogs.core;
+
+import com.whylogs.core.metrics.Metric;
+import com.whylogs.core.metrics.OperationResult;
+import com.whylogs.core.schemas.ColumnSchema;
+import com.whylogs.core.views.ColumnProfileView;
+import java.util.ArrayList;
+import java.util.HashMap;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public class ColumnProfile<T> {
```

Comment:
How does this work for columns of mixed types?

Reply:
You'd need to pick the top type so for int and float it would be Number. This is going to be true of the data they have in the row as well. My point was to keep it matched up with how they would be using it in Java.